astropy/convolution/convolve.py

```diff
@@ -546,6 +546,12 @@ def convolve_fft(array, kernel, boundary='fill', fill_value=0.,
     complex_dtype : numpy.complex, optional
         Which complex dtype to use.  `numpy` has a range of options, from 64 to
         256.
+    dealias: bool, optional
+        Default off. Zero-pad image to allow enable explicit dealiasing
+        of convolution. With ``boundary='wrap'``, this will be disabled.
+        Note that for an input of size N (entries) this will increase
+        the size of the computed convolution by at least 1.5*N. This
+        may result in significantly more memory usage.
```

Comment:
```suggestion
        Note that for an input of nd dimensions this will increase
        the size of the temporary arrays by at least ``1.5**nd``.
        This may result in significantly more memory usage.
```
Slight misunderstanding. I have switched to 'nd' which is the terminology at the top of the docstring.
The last line could perhaps better be added to the Notes following the one on `psf_pad` impact.